omega/omega-transaction/src/main/java/org/apache/servicecomb/saga/omega/transaction/TransactionAspect.java

```diff
@@ -57,7 +57,13 @@ Object advise(ProceedingJoinPoint joinPoint, Compensable compensable) throws Thr
     context.newLocalTxId();
 
     TimeAwareInterceptor interceptor = new TimeAwareInterceptor(this.interceptor);
-    interceptor.preIntercept(localTxId, signature, joinPoint.getArgs());
+    AlphaResponse response = interceptor.preIntercept(localTxId, signature, joinPoint.getArgs());
+    if (response.aborted()) {
+      String abortedLocalTxId = context.localTxId();
+      context.setLocalTxId(localTxId);
+      throw new OmegaTxAbortedException("Abort local sub transaction " + abortedLocalTxId +
```

Comment:
better to throw TransactionException defined in spring, because user is not going to catch OmegaTxAbortedException.

Reply:
using the spring's TransactionException will introduce spring dependency in our omega transaction module, is it all right to do so?